test/run.mjs

```diff
@@ -29,6 +29,19 @@ const SNAPSHOTS = path.join(__dirname, 'run-snapshots');
 /** @type {string | undefined} */
 const SUBCOMMAND = process.argv[2];
 
+const nodeVersionOutput = await $`node --version`;
+const nodeVersion = nodeVersionOutput.stdout.toString().slice(1).trim();
```

Comment:
Is there a reason this doesn't work?

```suggestion
const nodeVersion = process.verisons.node;
```

I'd be slightly concerned that some version managers might make shelling out return the wrong answer. Probably not though.

---

I also have a bit of a preference for normalizing the error message like in https://github.com/rtfeldman/node-test-runner/pull/635, but this is certainly better for now.

Reply:
Oh, yeah, maybe we should do that instead :+1:
I'll keep this for now but we can improve this later.